hub/apps/get-started/index.md

```diff
@@ -93,6 +93,30 @@ For more information about Windows Forms apps, see the following articles:
 * [API reference (.NET)](/dotnet/api/index)
 * [Enhancing Windows Forms apps](/dotnet/framework/winforms/advanced/)
 
+#### [React Native for Windows](#tab/rnw)
+[React Native](https://reactnative.dev) is a development platform from Facebook which allows building cross-platform apps.
+React Native for Windows brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you touse JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
```

Comment:
```suggestion
React Native for Windows brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you to use JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
```

Reply:
```suggestion
[React Native for Windows](https://aka.ms/reactnative) brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you touse JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
```
Not sure if this is the right place for _this_ link, but a link of some sort seems appropriate. Somewhere needs to link to the root aka.ms.